src/Components/Wishlist/wishlist.js

```diff
@@ -1,9 +1,44 @@
 import React from 'react'
+import { useAuth } from '../../Context/AuthContext/authContext'
+import { useCart } from '../../Context/cartContext';
+import { useWishlist } from '../../Context/wishlistContext';
 import './wishlist.css'
 
 const Wishlist = () => {
+  const {authState: {wishlist, cart}} = useAuth();
+  const {addToCart} = useCart();
+  const {removeFromWishlist} = useWishlist();
   return (
-    <div>Wishlist</div>
+    <section className="products" id="products">
+    <h1 className="heading" style={{marginTop: '8rem'}}> <span>Wishlist</span> </h1>
+    <div className='product-card'>
+    {wishlist.map((item)=>{
```

Comment:
can destructure items here so that you don't need to write it  multiple times
`{wishlist.map(({name,price,qty,etc})`